src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,19 @@
+package duke;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class TaskListTest {
+    @Test
+    public void test() throws DuckException {
```

Comment:
More descriptive test names would be preferable?
E.g. `unitBeingTested_descriptionOfTestInputs_expectedOutcome`